lib/Doctrine/ORM/Tools/ResolveTargetEntityListener.php

```diff
@@ -110,12 +109,9 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $args)
     }
 
     /**
-     * @param ClassMetadataInfo $classMetadata
-     * @param mixed[]           $mapping
-     *
-     * @return void
+     * @param mixed[] $mapping
      */
-    private function remapAssociation($classMetadata, $mapping)
+    private function remapAssociation(ClassMetadata $classMetadata, $mapping): void
```

Comment:
any reason why not declaring type for $mapping?

Reply:
That one slipped through :P